### PR TITLE
Add Kotlin property (KProperty0) based assertThat method

### PR DESCRIPTION
**Pull request by maverick1601**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,9 +43,15 @@ class PersonTest {
 
     @Test
     fun testAge() {
-        assertThat("age", person.age).isGreaterThan(20)
```
Comment (maverick1601) [inline README.md]: The order of arguments seemed to be wrong.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [0.13] - 2019-01-17
+## [0.13] - 2019-02-13
```
Comment (evant) [inline CHANGELOG.md]: 0.13 has already been released, please create a new [Unreleased] section for new changes.

File: README.md (hunk 1 above)
Comment (evant) [reply README.md]: Good catch! Forgot to update the README when we changed the order.

File: assertk-common/src/main/kotlin/assertk/assert.kt [hunk 3]
```diff
@@ -164,6 +165,20 @@ fun <T> assert(actual: T, name: String? = null): Assert<T> = assertThat(actual,
  */
 fun <T> assertThat(actual: T, name: String? = null): Assert<T> = ValueAssert(actual, name, null)
 
+/**
+ * Asserts on the given property value with an optional name.
```
Comment (evant) [inline assertk-common/src/main/kotlin/assertk/assert.kt]: Should better explain why you'd use this version, particularly that it uses the property's name by default.

Comment (maverick1601) [reply assertk-common/src/main/kotlin/assertk/assert.kt]: Changed the comment.

**Pull request merged**